### ISSUE TITLE
kata-containers-runtime-podman-clh-fedora-31 failure: Unable to find a match: podman-plugins-2:2.0.4-1.fc31.x86_64

### ISSUE THREAD

**Issue by fidencio** [kata-containers/tests#2946]
```
11:00:00 Fedora Modular 31 - x86_64                      6.8 MB/s | 5.2 MB     00:00    
11:00:02 Fedora Modular 31 - x86_64 - Updates            4.0 MB/s | 4.2 MB     00:01    
11:01:10 Fedora 31 - x86_64 - Updates                    415 kB/s |  27 MB     01:06    
11:01:23 Fedora 31 - x86_64                               13 MB/s |  71 MB     00:05    
11:01:42 Last metadata expiration check: 0:00:01 ago on Fri 02 Oct 2020 09:01:22 AM UTC.
11:01:47 Last metadata expiration check: 0:06:07 ago on Fri 02 Oct 2020 08:55:40 AM UTC.
11:01:49 Package glib2-2.62.6-2.fc31.x86_64 is already installed.
11:01:49 Package gpgme-1.13.1-7.fc31.x86_64 is already installed.
11:01:49 Package libassuan-2.5.3-2.fc31.x86_64 is already installed.
11:01:49 Package libgpg-error-1.36-2.fc31.x86_64 is already installed.
11:01:49 Package libseccomp-2.5.0-3.fc31.x86_64 is already installed.
11:01:49 Package libselinux-2.9-6.fc31.x86_64 is already installed.
11:01:49 Package ostree-libs-2020.3-4.fc31.x86_64 is already installed.
11:01:49 No match for argument: podman-plugins-2:2.0.4-1.fc31.x86_64
11:01:49 Error: Unable to find a match: podman-plugins-2:2.0.4-1.fc31.x86_64
11:01:49 ++ handle_error 69
11:01:49 ++ local exit_code=123
11:01:49 ++ local line_number=69
11:01:49 ++ echo 'Failed at /tmp/jenkins/workspace/kata-containers-runtime-podman-clh-fedora-31-PR/ci_entry_point.sh +69: .ci/jenkins_job_build.sh "${repo_to_test}"'
11:01:49 Failed at /tmp/jenkins/workspace/kata-containers-runtime-podman-clh-fedora-31-PR/ci_entry_point.sh +69: .ci/jenkins_job_build.sh "${repo_to_test}"
11:01:49 ++ exit 123
11:01:50 Build step 'Execute shell' marked build as failure
```

The CI failure is happening because the podman-plugins on Fedora 31 has been updated to `2:2.1.1-3`, and this should also be bumped on our scripts.

**Comment by ariel-adam** [comment]
@fidencio is this issue still happening from what you know? The merged PR seems to have fixed a different issues #2992

**Comment by fidencio** [comment]
From what I know, no, it's not happening anymore (but, hey, I haven't been following 1.x closely for some time).
I'll take the freedom to close this one and re-open it in case we face the issue again.
